tools/sigclient/functionalQueries/eval_groupby_2col_2stats.json

```diff
@@ -2,7 +2,7 @@
     "queryText": "city=Boston | eval temp_value = http_status%2 | stats count, sum(latitude) BY gender, temp_value",
     "expectedResult": {
          "totalMatched": {
-             "value": 984,
+             "value": 4,
              "relation": "eq"
```

Comment:
Is this the desired behavior? 4 is the number of rows returned, but I assume 984 is the number of records matching the search `city=Boston` (or maybe 984 came from somewhere else?)

cc: @nkunal